### PR TITLE
Technical Steering Committee Charter

### LINKED ISSUE #815: Technical Steering Committee (TSC) Charter
As part of the Academy Software Foundation graduation process, OpenTimelineIO needs to adopt a charter for our technical steering committee, which includes a governance model.

Here is the ASWF TSC charter template as a starting point:
https://github.com/AcademySoftwareFoundation/tac/blob/master/process/charter_template.md

### PR DESCRIPTION
**Pull request by jminor**
```Fixes #815```

This PR adds the **proposed** TSC Charter, based on the standard ASWF TSC charter template, with changes specific to OpenTimelineIO.

There are still some changes & decisions that need to be made before this is approved, notably the CONTRIBUTING.md, list of TSC members, etc.

### CODE REVIEW

**Comment by jminor** [comment]
Also note, that I separated each of our changes from the ASWF template into separate commits so we can review & discuss those independently.

File: tsc/OpenTimelineIO_TSC_Charter.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  * **b.** Quorum for TSC meetings requires at least two-thirds of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met, but will be prevented from making any decisions at the meeting.
+  * **c.** Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
+  * **d.** In the event a vote cannot be resolved by the TSC, any voting member of the TSC may refer the matter to the Series Manager or its designee for assistance in reaching a resolution.
+
+## 4. Compliance with Policies
+  * **a.** This Charter is subject to the Series Agreement for the Project and the Operating Agreement of LF Projects. Contributors will comply with the policies of LF Projects as may be adopted and amended by LF Projects, including, without limitation the policies listed at https://lfprojects.org/policies/.
```
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:37]: The originating boilerplate is from the LF. I wonder if we were meant to /s/LF/ASWF/ in section 4?

File: tsc/OpenTimelineIO_TSC_Charter.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  * **e.** The Project will operate in a transparent, open, collaborative, and ethical manner at all times. The output of all Project discussions, proposals, timelines, decisions, and status should be made open and easily visible to all. Any potential violations of this requirement should be reported immediately to the LF Projects Manager.
+
+## 5. Community Assets
+  * **a.** LF Projects shall hold title to all trade or service marks used by the Project ("Project Trademarks"), whether based on common law or registered rights. Project Trademarks shall be transferred and assigned to LF Projects to hold on behalf of the Project. Any use of any Project Trademarks by participants in the Project shall be in accordance with the license from LF Projects and inure to the benefit of LF Projects.
```
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:44]: In our "steps to graduate" checklist, we note transferring trademarks to the ASWF, not the LF. Does section 5 require a /s/LF/ASWF/ ?

File: tsc/OpenTimelineIO_TSC_Charter.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  * **b.** The Project shall, as permitted and in accordance with such license from LF Projects, develop and own all Project GitHub and social media accounts, and domain name registrations created by the Project community.
+  * **c.** Under no circumstances shall LF Projects be expected or required to undertake any action on behalf of the Project that is inconsistent with the tax-exempt status or purpose, as applicable, of LFP, Inc. or LF Projects, LLC.
+
+## 6. General Rules and Operations.
+  * **a.** The Project will:
```
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:49]: Ditto /s/LF/ASWF/ ?

File: tsc/OpenTimelineIO_TSC_Charter.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  * **c.** If an alternative inbound or outbound license is required for compliance with the license for a leveraged open source project or is otherwise required to achieve the Project's mission, the Governing Board of the Academy Software Foundation of The Linux Foundation ("Governing Board") or the Governing Board's designated committee may approve the use of an alternative license for specific inbound or outbound contributions on an exception basis. Any exceptions must be approved by a vote of the Governing Board and must be limited in scope to what is required for such purpose. To request an exception, please describe the contribution, the alternative open source license(s), and the justification for using an alternative open source license for the Project.
```
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:63]: is it a sentence-of-meaning to say "the Academy Software Foundation of the The Linux Foundation" or is this a search and replace glitch?

I think we can omit the word "please" here, as we're not making a request, we're just providing instruction.

File: tsc/OpenTimelineIO_TSC_Charter.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  * **d.** Contributed files should contain license information, such as SPDX short form identifiers, indicating the open source license or licenses pertaining to the file.
```
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:64]: do we want to "such as" SPDX? I think it is better to just say, "either a SPDX short form identifier, or the full text of the license if the license is not a standard license"

File: tsc/OpenTimelineIO_TSC_Charter.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+## 8. Amendments
+  * **a.** This charter may be amended by a two-thirds vote of the entire TSC and is subject to approval by LF Projects.
```
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:67]: Is it subject to approval by LF Projects? Or the ASWF TAC? or ASWF Projects?

File: tsc/OpenTimelineIO_TSC_Charter.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+# Technical Charter (the "Charter") for OpenTimelineIO a Series of LF Projects, LLC
+
+This charter (the "Charter") sets forth the responsibilities and procedures for technical contribution to, and oversight of, the OpenTimelineIO project, which has been established as OpenTimelineIO a Series of LF Projects, LLC (the "Project").  LF Projects, LLC ("LF Projects") is a Delaware series limited liability company. All Contributors to the Project must comply with the terms of this Charter.
```
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:3]: Is this LF reference correct?

**Review by meshula — COMMENTED** [review]
My first observation is that there's a lot of mention of LF and not ASWF. Is that correct? Or is that an artifact of starting with LF boilerplate? My gut says its an artifact.

File: tsc/OpenTimelineIO_TSC_Charter.md (hunk 7 above)
Comment (meshula) [inline tsc/OpenTimelineIO_TSC_Charter.md:3]: My confusion around the charter would be resolved by declaring the relationship between the ASWF and the LF for the purposes of the charter. For example, 

ASWF ("The Academy Software Foundation") is a group of projects within the LF Projects.

**Comment by jminor** [comment]
Closing this PR in favor of this one:
https://github.com/AcademySoftwareFoundation/foundation/pull/6

Let's move our discussion & questions over there instead.

**Pull request closed without merging**